Description: fix-japanese-input-problem
 Put the rest of  override redirect window types into the top window group,
 fix Japanese Input Problem.
Author: luyikei <luyikei.qmltu@gmail.com>
Origin: upstream
Forwarded: not needed
Applied-Upstream: https://github.com/linuxmint/muffin/commit/baba0ed
Last-Update: 2013-04-06
--- a/src/compositor/meta-window-actor.c
+++ b/src/compositor/meta-window-actor.c
@@ -1513,7 +1513,10 @@
   
   if (window->type == META_WINDOW_DROPDOWN_MENU ||
       window->type == META_WINDOW_POPUP_MENU ||
-      window->type == META_WINDOW_COMBO){
+      window->type == META_WINDOW_COMBO ||
+      window->type == META_WINDOW_NOTIFICATION ||
+      window->type == META_WINDOW_DND ||
+      window->type == META_WINDOW_OVERRIDE_OTHER){
     clutter_container_add_actor (CLUTTER_CONTAINER (info->top_window_group),
 			       CLUTTER_ACTOR (self));
   }
